packages/components/autocomplete/src/autocomplete.vue

```diff
@@ -194,12 +195,16 @@ const handleInput = (value: string) => {
   emit('input', value)
   emit(UPDATE_MODEL_EVENT, value)
   suggestionDisabled.value = false
-  activated.value = Boolean(value)
+  activated.value ||= isClear && Boolean(value)
```

Comment:
**Question:** can we do something like:

```ts
const valuePresented = Boolean(value)
```
?

Reply:
ok, and i update [`autocomplete examples`](docs/examples/input/autocomplete.vue)